NuGet.config.orig

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
orig files got checked in here

Reply:
@sfoslund  This should be fixed, could you re-review?